src/main/java/com/example/GymProject/config/AppConfig.java

```diff
@@ -0,0 +1,10 @@
+package com.example.GymProject.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = "com.example")
+public class AppConfig {
```

Comment:
Please add the following here to resolve @Value in other component properly:
`@PropertySource("classpath:application.properties")`